### DISCUSSION TITLE
Call validators after object creation

### DISCUSSION THREAD

**Discussion by MVrachev** [pydantic/pydantic#2539] [Question]
Hi, is there a way to call class attributes (fields) validators after an instance was created?

An object could do many things during its life and the state of the fields can easily be modified to not valid.

Then, is there a way for a user of pydantic to call all field validators (or part of them) just before he/she do something sensitive as
writing to a database?

**Comment by PrettyWood** [comment]
Hi @MVrachev 
Would https://github.com/samuelcolvin/pydantic/issues/1864#issuecomment-679044432 help? :)

**Reply by MVrachev** [reply]
It will work, but as I explained in my comment here https://github.com/samuelcolvin/pydantic/issues/668#issuecomment-801927266 until this function is not documented as public API, I am not sure how safe it's for use.
